scripts/package_lambdas.sh

```diff
@@ -0,0 +1,12 @@
+pushd venvpy/lib/python3.7/site-packages && zip -r9 base_function.zip .
+popd
+
+cp venvpy/lib/python3.7/site-packages/base_function.zip lambdas/submit_job/function.zip
+cp venvpy/lib/python3.7/site-packages/base_function.zip lambdas/upload_results_to_datasets/function.zip
+cp venvpy/lib/python3.7/site-packages/base_function.zip lambdas/check_datasets_saved/function.zip
+
+zip -g lambdas/submit_job/function.zip lambdas/submit_job/function.py
+zip -g lambdas/upload_results_to_datasets/function.zip lambdas/upload_results_to_datasets/function.py
+zip -g lambdas/check_datasets_saved/function.zip lambdas/check_datasets_saved/function.py
+
+rm venvpy/lib/python3.7/site-packages/base_function.zip
```

Comment:
while this might work here, it is better to do this inside a docker container using an AWS image, to ensure all dependencies are met.

You can use this dockerfile as a template. Every lambda function will need its own Dockerfile which should live inside the function folder.
https://github.com/wri/gfw-raster-analysis-lambda/blob/master/Dockerfile.

in the deploy script you can then build the container, copy the file to host and upload it to AWS:
```
docker build -t  gfw/<my_lambda_function>
docker create -ti --name lambda gfw/<my_lambda_function> bash
docker cp dummy:/path/to/file /dest/to/file
docker rm -f lambda
```

Reply:
Fixed in #5 and #6